doc/pbh/pbh-design.md

```diff
@@ -569,23 +818,27 @@ ip-mask    = ipv4-addr / ipv6-addr
 
 ## 2.5 Flows
 
-### 2.5.1 PBH add
+### 2.5.1 Key modification
+
+#### 2.5.1.1 PBH add
 
 ![PBH add flow](images/pbh_add_flow.svg "Figure 3: PBH add flow")
 
 ###### Figure 3: PBH add flow
 
-### 2.5.2 PBH update
+#### 2.5.1.2 PBH remove
 
-![PBH update flow](images/pbh_update_flow.svg "Figure 4: PBH update flow")
+![PBH remove flow](images/pbh_remove_flow.svg "Figure 4: PBH remove flow")
 
-###### Figure 4: PBH update flow
+###### Figure 4: PBH remove flow
 
-### 2.5.3 PBH remove
+### 2.5.2 Field modification
 
-![PBH remove flow](images/pbh_remove_flow.svg "Figure 5: PBH remove flow")
+#### 2.5.2.1 PBH update
 
-###### Figure 5: PBH remove flow
+![PBH update flow](images/pbh_update_flow.svg "Figure 5: PBH update flow")
```

Comment:
The flow diagram mentions mgmt. framework. Are we targetting it? Is there a difference when user types in CLI commands?
The flow also mentions transactions. Could you please clarify the underlaying mechanism, is this a redis transaction? Is it done on top of swss-common or direct redis calls?
In case we do have redis transactions - are they required in case we do not support field set and delete in a single CLI update command?

Reply:
done